designer/apps/api/v1/serializers.py

```diff
@@ -37,29 +37,45 @@ def get_branding(self, obj):
         if not branding:
             return {}
 
-        return {
-            'cover_image': branding.cover_image.file.url,
-            'texture_image': branding.texture_image.file.url,
+        cover_image = getattr(branding, 'cover_image', None)
+        texture_image = getattr(branding, 'texture_image', None)
+        banner_background_color = getattr(branding, 'banner_background_color', None)
+
+        serialized_branding = {
             'organization_logo': {
                 'url': branding.organization_logo_image.file.url,
                 'alt': branding.organization_logo_alt_text,
             },
             'banner_border_color': branding.banner_border_color,
         }
 
+        if cover_image:
+            serialized_branding.update({
+                'cover_image': cover_image.file.url,
+            })
+        if texture_image:
+            serialized_branding.update({
+                'texture_image': texture_image.file.url,
```

Comment:
Will the `.file.url` always be available if `cover_image` or `texture_image` is available from the branding?

Reply:
Yes, those fields are foreign keys to wagtail's image model.